deepcell/image_generators/semantic.py

```diff
@@ -304,21 +304,14 @@ def flow(self,
 
         Args:
             train_dict (dict): Consists of numpy arrays for X and y.
-            batch_size (int): Size of a batch.
-            skip (int): Number of skip connections to yield data.
+            batch_size (int): Size of a batch. Defaults to 1.
             shuffle (bool): Whether to shuffle the data between epochs.
+                Defaults to True
             seed (int): Random seed for data shuffling.
-            save_to_dir (str): Optional directory where to save the pictures
-                being yielded, in a viewable format. This is useful
-                for visualizing the random transformations being
-                applied, for debugging purposes.
-            save_prefix (str): Prefix to use for saving sample
-                images (if save_to_dir is set).
-            save_format (str): Format to use for saving sample images
-                (if save_to_dir is set).
```

Comment:
Let's keep these docstrings around as they are still valid inputs for the class. I think we should put them in the MovieGenerator as well.